webapp/dataclass.py

```diff
@@ -0,0 +1,22 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class AssetSearchParams:
+    tag: str = ""
+    asset_type: str = ""
+    product_types: list = None
```

Comment:
```suggestion
    tag: str = ""
    asset_type: str = ""
    product_types: list = None
```
Why are we treating the list of regular tags differently to the list of product tags?

Reply:
Also, I think we should be more consistent with our naming of the tags. From what I understand from the figma, there are two types of tag: 'product' and 'other'. The names should reflect this

Reply:
I guess I am missing some context, but why are we not allowing search for multiple tags?